pkg/apis/enterprise/v1/common_types.go

```diff
@@ -178,36 +188,49 @@ type IndexAndCacheManagerCommonSpec struct {
 	HotlistBloomFilterRecencyHours uint `json:"hotlistBloomFilterRecencyHours,omitempty"`
 }
 
+// AppSourceDefaultSpec defines config common for defaults and App Sources
+type AppSourceDefaultSpec struct {
+	// Remote Storage Volume name
+	VolName string `json:"volumeName,omitempty"`
+
+	// Scope of the app deployment: cluster, local. Scope determines the App deployment path on the splunk enterprise pod. The path is dependent upon the Kind defined in the CRD
```

Comment:
// Scope of the App deployment: cluster, local. Scope determines whether the App(s) are installed locally or cluster-wide